circle.yml

```diff
@@ -386,7 +386,7 @@ commands:
         type: string
     steps:
       - restore_cached_workspace
-      - run: yarn workspace @packages/launchpad cypress:run --browser <<parameters.browser>>
+      - run: yarn workspace @packages/launchpad build && yarn workspace @packages/launchpad cypress:run --browser <<parameters.browser>>
```

Comment:
Turns out `build` is not necessary here, I was wrong about the root cause of the flake.